lib/gulp-tasks/patterns-import.js

```diff
@@ -0,0 +1,73 @@
+/**
+ *  @fileOverview
+ * Uses Gulpjs to import html patterns
+ *
+ *  @author       Scott Nath
+ *
+ *  @requires     NPM:gulp
+ *  @requires     NPM:lodash.merge
+ *  @requires     NPM:pattern-importer
+ */
+'use strict';
+
+var gulp = require('gulp'),
+  merge = require('lodash.merge'),
+  utility = require('../utility');
+
+/**
+ * Function to get default options for an implementation of patternlab-import
+ * @returns {Object}  options  an object of default patternlab-import options
+ */
+function getDefaultOptions () {
```

Comment:
@scottnath We have getDefaultOptions function all over the global scope. If they are local to their respective files/modules, please contain them within functions or within some sort of localized scope.